### ISSUE TITLE
NoSuchElementException when parsing JUnit report without type

### ISSUE THREAD

**Issue by uhafner** [tomasbjerre/violations-lib#97]
The JUnit parser accesses an optional without guard (if neither message or type is filled). 
```
final String message = messageOpt.orElse(findAttribute(failure, "type").get());
```
See original bug report in https://issues.jenkins-ci.org/browse/JENKINS-63527.
There is also https://github.com/jenkinsci/analysis-model/pull/500 that exposes the bug

**Comment by tomasbjerre** [comment]
Thanks for reporting. Released a fix in 1.122.

**Comment by uhafner** [comment]
Thanks for fixing so fast!
